packages/ui/icons/Passport.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
💭 My bad as I didn't unify icon names on Figma. Now it's fixed. Could you export them to the project again? Also as those are not normal icons we use across the app I think it would be great to put them in the `ui/icons/technologies` directory.